docs/repositories/software_eessi_io.md

```diff
@@ -0,0 +1,177 @@
+## EESSI software stack (2023.06)
+
+### Reporting problems
+
+If you notice any problems, please report them via https://gitlab.com/eessi/support or send it in via email, `support (at) eessi.io`.
```

Comment:
```suggestion
## Question or problems?

If you have any questions regarding EESSI, or if you experience a problem in accessing or using it,
please [open a support request](../support.md).

Reply:
No mention that they can email us as well?